workflow-templates/im-run-annotate-pagerduty.yml

```diff
@@ -0,0 +1,85 @@
+#    Creates an ad hoc PagerDuty event annotation for a specified
+#    service when someone kicks it off manually.
+#
+# Frequency:
+#    - This workflow should only be used once per repository assuming
+#      there is only one app insights instance set up per env
+#
+# Projects to use this Template with:
+#    - Azure App Service or Function (Optional Template)
+#    - Azure SQL Database            (Optional Template)
+#    - Terraform                     (Optional Template)
+#
+# TODO: Prerequisites:
+#    - Ensure each of the repo-level and env-level secrets used in this workflow have been populated by an admin in your repository.
+
+name: Runbook - Annotate PagerDuty
+run-name: Annotate ${{ inputs.eventName }} in ${{ inputs.environment }} PagerDuty
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: The environment the service is in
+        required: true
+        default: prod
+        type: choice
+        options: # TODO: Update for the environments that are available
+          - dev
+          - qa
+          - stage
+          - demo
+          - uat
+          - prod
+      service-ids:
+        description: Comma delimited list of service IDs to annotate
+        required: true
+        
+      event:
+        description: The event description to record
+        required: true
+
+env:
+  ENVIRONMENT: ${{ github.event.inputs.environment }}
+
+jobs:
+  create-annotation:
+    runs-on: [self-hosted, im-linux]
+    environment: ${{ github.event.inputs.environment }}
+    steps:
+      - run: |
+          echo $'
+          | Build Arguments | Value                                      |
+          | ---             |  ---                                       |
+          | Environment     | `${{ env.ENVIRONMENT }}`                   |
+          | Event           | `${{ github.event.inputs.event }}`         |' >> $GITHUB_STEP_SUMMARY
+              
+      # For more information and best practices on the usage and options available
+      # for this action go to: https://github.com/im-open/set-environment-variables-by-scope#usage-instructions
+      - name: Set Variables
+        id: set-variables
+        uses: im-open/set-environment-variables-by-scope@v1.1
+        with:
+          scope: ${{ github.event.inputs.environment }}
+        env:
+          # This variable is used to upload and download blobs from blob storage
+          RESOURCE_GROUP@dev: ''
+          RESOURCE_GROUP@qa: ''
+          RESOURCE_GROUP@stage: ''
+          RESOURCE_GROUP@demo: ''
+          RESOURCE_GROUP@uat: ''
+          RESOURCE_GROUP@prod: ''
+          # TODO: For the following app insights name inputs, fill in the value if you have the environment and delete the environment if it does not exist
+          APP_INSIGHTS_NAME@dev: ''
+          APP_INSIGHTS_NAME@qa: ''
+          APP_INSIGHTS_NAME@stage: ''
+          APP_INSIGHTS_NAME@demo: ''
+          APP_INSIGHTS_NAME@uat: ''
+          APP_INSIGHTS_NAME@prod: ''
```

Comment:
It doesn't look like RESOURCE_GROUP or APP_INSIGHTS is used, so this step can probably be removed.
```suggestion
```